src/ontology/mondo-edit.obo

```diff
@@ -291102,6 +291584,7 @@ subset: ordo_malformation_syndrome {source="Orphanet:96170"}
 subset: orphanet_rare {source="Orphanet:96170"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "22) SYNDROME" EXACT ABBREVIATION [OMIM:609029]
```

Comment:
The OMIM content is not being parsed correctly for https://omim.org/entry/609029 so the value `22)` is incorrectly represented as a synonym in omim.owl